kupyna/Cargo.toml

```diff
@@ -0,0 +1,31 @@
+[package]
+name = "kupyna"
+version = "0.1.0"
+description = "Kupyna (DSTU 7564:2014) hash function"
+authors = ["RustCrypto Developers"]
+license = "MIT OR Apache-2.0"
+readme = "README.md"
+edition = "2021"
+documentation = "https://docs.rs/kupyna"
+repository = "https://github.com/RustCrypto/hashes"
+keywords = ["crypto", "hash", "kupyna"]
+categories = ["cryptography", "no-std"]
+rust-version = "1.71"
+
+[dependencies]
+digest = "=0.11.0-pre.10"
+hex-literal = "0.4"
+
+[dev-dependencies]
+digest = { version = "=0.11.0-pre.10", features = ["dev"] }
+hex-literal = "0.4"
+base16ct = { version = "0.2", features = ["alloc"] }
+
+[features]
+default = ["std"]
+std = ["digest/std"]
+zeroize = ["digest/zeroize"]
+
+[package.metadata.docs.rs]
+all-features = true
+rustdoc-args = ["--cfg", "docsrs"]
```

Comment:
We no longer need to set `rustdoc-args` since this configuration flag is passed by default by docs.rs build system.